tests/utils/test_integration.py

```diff
@@ -10,8 +10,9 @@
 from datetime import timedelta
 
 import pytest
+from watchfiles import Change
```

Comment:
This should never be imported in integration tests, as we shouldn't expose `watchfiles` to the end user.

Reply:
you're absolutely right. I should have used FileWatcher.EventType instead. Will update it